sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -104,6 +120,21 @@ def __init__(
         self.needs_X = self.algorithm_exogenous_support and bool(self.futr_exog_list)
 
         self.set_tags(**{"ignores-exogeneous-X": not self.needs_X})
+        if self.broadcasting:
+            self.set_tags(
+                **{
+                    "y_inner_mtype": "pd.Series",
+                    "X_inner_mtype": "pd.DataFrame",
+                    "capability:global_forecasting": False,
+                }
+            )
+        from warnings import warn
```

Comment:
I think this is recommended by @@fkiraly:

```python3
from sktime.utils.warnings import warn
```

Reply:
updated